markdown/manual.md

```diff
@@ -94,127 +94,168 @@ Annual cost of living increases will be combined with any merit increases descri
+We expect everyone to be able to get their work done in a timely fashion, and to not have to work on personal time.  Rested and refreshed people are better team members, and we strongly encourage enjoying life outside of the office.  
 
-To schedule time off, permission must be granted by your supervisor no less than 5 business days in advance, the dates and location of your travel must be put onto the team calendar, and the team should be notified via email.  To track time off, please track the vacation time in Harvest.
+*Employees should take a minimum of two weeks of paid vacation per year, totaling 10 business days.*
 
-For personal, medical, or family leave, each employee is encouraged to take the current Family and Medical Leave Act maximum.
+For personal, medical, or family leave, each employee is encouraged to take the current [Family and Medical Leave Act](https://www.dol.gov/general/topic/benefits-leave/fmla) maximum.
 
 For maternity and paternity leave, employees are encouraged to take up to 16 weeks of leave.
 
+#### Taking Leave
 Employees must inform your supervisor in writing as far in advance as is reasonable for staff and project planning purposes.  All extended periods of leave must be discussed with the Executive Director.
 
+To schedule time off, permission must be granted by your supervisor no less than 5 business days in advance, the dates your travel must be put onto the team calendar, and the team should be notified via email.
+
 ### Sick Days
 Please notify the team if you are sick and need to take the day off. Even if you are feeling well enough to work, please work from home until you are completely recovered, to avoid spreading illness through our small office.
 
-
 ## How We Work
 
 ### Email Accounts, Document Sharing & Password Sharing
-We will set you up with an organization Gmail account.  This will provide you access to our email, calendar, and Google+ accounts.  Please use this for all work-related correspondence, scheduling, etc.  Personal accounts should not be used for work-related activities.
+We will set you up with an organizational Google Suite account.  This will provide you access to all the basic necessities like email, calendar, and shared storage.  
+
+Please use your organizational email address for all work-related correspondence, scheduling, etc.  Personal accounts should not be used for work-related activities.
 
-You will be granted access to our shared passwords in Passpack.  All passwords that need to be shared with the organization must be shared using Passpack, you must not share them via email or other insecure methods.
+You will be granted access to our shared passwords in Passpack.  All passwords that need to be shared with the organization must be shared using Passpack or a secure service such as [Toaster](https://fugacious.18f.gov/). Passwords should not be shared via email or other insecure methods.
```

Comment:
What if we had a list of tools that are used for certain tasks (like password managing), and throughout the text we just reference "our password manager" with a link to the list. That way if a tool changes, we don't have to hunt down all references to it?

Reply:
I like it, going to take it a step further and try just a general discussion of tooling in the HR manual, then move the more detailed tooling discussion elsewhere...